test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -4634,6 +4634,33 @@ def forward(self, x):
                           input_names=["x"],
                           dynamic_axes={"x": {0: "batch_size", 1: "dims"}})
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    @disableScriptTest()
+    def test_dynamic_chunk(self):
+        class ChunkModel(torch.nn.Module):
+            def __init__(self, dim=1):
+                super(ChunkModel, self).__init__()
+                self.dim = dim
+
+            def forward(self, x):
+                return torch.chunk(x, x.size(0), dim=self.dim)
+
+        model = ChunkModel()
+        model.eval()
+        model_neg_dim = ChunkModel(-1)
+        model_neg_dim.eval()
+        x = torch.randn(3, 18)
+
+        for dim_size_ in range(13, 16):
```

Comment:
Why can't we use a 3 dimensions here?

Reply:
Could you elaborate? I'm not sure what you mean by 3 dimensions

Reply:
> Could you elaborate? I'm not sure what you mean by 3 dimensions

I see the code was changed from "for dim_size_ in range(12, 24, 3):"  to "for dim_size_ in range(13, 16):". So I'm a little curious about why this happened. I think (12, 24, 3) will cover more cases.

Reply:
So, the reason I changed steps from 3 to 1, is that range(12, 24, 3) only checks for cases which are divisible by chunk size 3 (12,15,18,21). Having it range from 13-16 with step=1, allows to test with both divisible and indivisible cases